pallets/parachain-staking/src/weights.rs

```diff
@@ -478,45 +444,45 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking AtStake (r:0 w:1)
 	#[rustfmt::skip]
 	fn select_top_candidates(x: u32, y: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 90_000
-			.saturating_add(Weight::from_ref_time(19_130_000 as u64).saturating_mul(x as u64))
-			// Standard Error: 90_000
-			.saturating_add(Weight::from_ref_time(6_041_000 as u64).saturating_mul(y as u64))
+		Weight::from_ref_time(44_110_000 as u64)
```

Comment:
This increase quite significantly as well.

Reply:
Same as above, the overhead might come from #1828